OpenSim/Examples/DataTable/example2.cpp

```diff
@@ -0,0 +1,82 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  example2.cpp                          *
```

Comment:
> I like different files because they can stand independently and users can just copy the whole file if they want to play with it.

I agree if they are distinct. The fact that the first is a TimeSeriesTable and the other DataTable seems like a minor difference for demonstrating how to populate a DataTable. I would argue in that case that it makes more sense for the first example to populate a simple DataTable and not a TimeSeriesTable and subsequent examples should demonstrate more complex table types.